src/components/MediumFeed/index.js

```diff
@@ -0,0 +1,69 @@
+import React, {useEffect, useState} from "react"
+import PropTypes from "prop-types"
+import FeedItemSmall from './FeedItemSmall'
+import FeedItem from './FeedItem'
+import "./style.sass"
+
+const MediumFeed = ({title, mediumUrl, small, limit=100}) => {
+
+  const [loading, setLoading] = useState(true)
+  const [feed, setFeed] = useState(null)
+
+  useEffect(() => {
+    getFeed()
+  }, [])
+  
+  const getFeed = () => {
+    const feedFetchUrl = `https://api.rss2json.com/v1/api.json?rss_url=${mediumUrl}`
+    setLoading(true)
+    fetch(feedFetchUrl).then((res) => res.json()).then((data) => {
+      setFeed(data)
+      setLoading(false)
+    })
+    .catch((err) => { throw err });
```

Comment:
Don't we have to `setLoading(false)` here also?

Reply:
In this case, Yes